lib/reel/session/crypto.rb

```diff
@@ -14,10 +14,10 @@ def encrypt val
         cipher = OpenSSL::Cipher::AES128.new :CBC
         cipher.encrypt
         # getting config depending on call from session/store
-        config = Reel::Session.configuration
+        config = @config || self.session_config
```

Comment:
So you are suggesting, we should pass `config` as parameter while calling `encrypt/decrypt` ..
Something like this => `Reel::Session::Crypto.encrypt(val,config)` ?